build.sbt

```diff
@@ -15,15 +16,15 @@ val akkaV = "2.4.11" // Note: akka-http branches from akkaV after 2.4.11
 // logBuffered in Test := false
 
 libraryDependencies ++= Seq(
-  "com.github.kxbmap" %% "configs" % "0.4.2",
+  "com.github.kxbmap" %% "configs" % "0.4.4",
   "com.typesafe" % "config" % "1.3.0",
-  "org.typelevel" %% "cats" % "0.7.2"
+  "org.typelevel" %% "cats" % "0.9.0"
```

Comment:
Yeah I knew that was coming and am also pleasantly surprised that's the only substantive change required, at least in Centaur-land.